### DISCUSSION TITLE
Error message showed up when importing qiime2 artifact

### DISCUSSION THREAD

**Discussion by SchuShoe** [micom-dev/micom#74] [Q&A] [answered]
"_Semantic type MicomMedium[Global] does not have a compatible directory format_"
This error message showed up when importing the data. 
Does anyone know how to fix it? 
![image](https://user-images.githubusercontent.com/101419498/159680262-3114b21b-d629-44cf-9aa8-7e5224126710.png)

Thanks in advance :D

**Comment by cdiener** [comment, marked as answer]
Do you have [q2-micom](https://github.com/micom-dev/q2-micom) installed in the Qiime 2 environment? You can check by running `!qiime micom --help` in the notebook. If you only want to use the Python micom package without Qiime you don't really need to save it as a Qiime artifact. Just saving the medium as CSV with `completed.to_csv("my_medium.csv", index=False)` and reading it with pandas before you use it (`medium = pd.read_csv("my_medium.csv")`) would be sufficient in that case.

**Reply by SchuShoe** [reply]
Thanks, the problem has been solved.
